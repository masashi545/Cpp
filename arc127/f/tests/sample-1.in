5
3 5 0 5
1 2 5 10
5 8 4 9
10 99 48 106
500000000 500000001 123456789 900000000
