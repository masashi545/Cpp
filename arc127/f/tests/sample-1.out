4
11
4
10
800000002
